rcongui_public/src/components/game/statistics/game-table.tsx

```diff
@@ -88,41 +104,30 @@ export function DataTable<TData extends Player, TValue>({columns, data, tableId}
                 <SelectValue placeholder={t('playersTable.team')}/>
               </SelectTrigger>
               <SelectContent>
-                <SelectItem value="all">{t('onlineStatusFilter.all')}</SelectItem>
-                {teamOptions.map(option =>
+                <SelectItem value="all">
+                  <div className={"pl-5"}>
+                    {t('onlineStatusFilter.all')} ({data.length})
+                  </div>
+                </SelectItem>
+                {teamOptions.map((option, index) =>
                   <SelectItem value={option}>
                     <div className="flex">
                       <TeamIndicator team={option as TeamEnum} className="block m-auto"/>
-                      <div className="pl-3">{t(option)}</div>
+                      <div className="pl-3">{t(option)} ({teamCounts[index]})</div>
                     </div>
                   </SelectItem>
                 )}
               </SelectContent>
             </Select>
           )}
           {table.getColumn('player') && (
-            <Fragment>
-              <Input
-                className="grow max-w-60 border shadow rounded"
-                onChange={({target}) => {
-                  setFilter(target.value);
-                }}
-                onKeyUp={({key}) => {
-                  if (key !== 'Enter' || filter === '' || filter === undefined) {
-                    return;
-                  }
-                  const orig = table.getColumn('player')?.getFilterValue() as [] ?? []
-                  table.getColumn('player')?.setFilterValue([...orig, filter]);
-                  setFilter('');
-                }}
-                placeholder={`${t('searchPlayer')}...`}
-                value={filter}
-                type="text"
-              />
-              <TagList
-                value={((table.getColumn('player')?.getFilterValue() as []) ?? [])}
-                onChange={(v) => table.getColumn('player')?.setFilterValue(v)}/>
-            </Fragment>
+            <SelectBox
+              options={playerFilterOptions}
+              multiple
+              value={playerFilter}
+              onChange={(values) => Array.isArray(values) && setPlayerFilter(values)}
+              placeholder={`${t('searchPlayer')}...`}
```

Comment:
Missing `placeholderInput`

Reply:
It's breaking out of the layout on smaller screens.
I know this may not be an app heavily used on mobile phones but it would be nice to handle this somehow.
Perhaps use `useMediaQuery` and if small screen, limit visible tags to 1 & set overlay hidden.

![image](https://github.com/user-attachments/assets/ad8af1c4-4aa0-42c2-83cf-124fd221bdc9)

Reply:
inputPlaceholder default value is "Search...". I changed it before to the same value as for the placeholder, but this doesnt look good/right.

Regarding the breaking layout: I tested the layout before i created the MR and i cant recreate the behaviour from the screenshot. I used chrome dev tools to test mobile resolutions. Its also weird that you still have the background image visible as this is the first element, which isn't visible anymore when you reduce the resolution. Do you have a local change?

I still added your suggestion with the limited tags for mobile as the bigger tag field lead to the popupover to be outside the screen.